packages/workflow/src/records/fhir.ts

```diff
@@ -1173,7 +1167,7 @@ export function createCorrectionRequestTask(
 function createNewTaskResource(
   previousTask: SavedTask,
   newExtensions: Extension[],
-  practitionerId: Practitioner['id'],
+  practitionerId?: Practitioner['id'],
```

Comment:
Is there a need to even keep this argument? Like we only seem to be passing "undefined" as the parameter

Reply:
Yeah for download, unassign, view endpoint's record, where we don't merge the inputBundle with heath response.

Reply:
Or we could use the `withPractitionerDetails` for them as well and remove the practitionerId param from here.